nutils/function.py

```diff
@@ -2452,6 +2452,34 @@ def partition(f: IntoArray, *levels: float) -> Sequence[Array]:
   steps = map(subtract, signs[:-1], signs[1:])
   return [.5 - .5 * signs[0]] + [.5 * step for step in steps] + [.5 + .5 * signs[-1]]
 
+def heaviside(f: IntoArray):
+  '''Create a heaviside step-function based on a scalar function f.
+
+  .. math:: H(f) &= 0     && f < 0
+
+            H(f) &= 0.5   && f == 0
```

Comment:
In math mode `==` seems non-standard. I suggest to replace `==` with `=`.

Reply:
done